source/routing/next_stop_time.h

```diff
@@ -216,9 +234,17 @@ struct CachedNextStopTime {
                    const bool clockwise) const;
 
 private:
-    using DtSt = std::pair<DateTime, const type::StopTime*>;
-    IdxMap<JourneyPatternPoint, std::vector<DtSt>> departure;
-    IdxMap<JourneyPatternPoint, std::vector<DtSt>> arrival;
+    struct Fun {
```

Comment:
I think it deserve a better name :wink: 

CacheCreator ? (not sure I correctly understand it's use though)